docs/modules/ROOT/pages/index.adoc

```diff
@@ -15,12 +23,32 @@ We essentially need to support two fairly distinct types of user:
 
 == Caveats and restrictions
 
-* Limited compatibility by supporting migrating from previous version to latest version. Best efforts will be made but the intention is this iterates often and no backwards compatibility is provided. We will show how to migrate from X-1 to X but no more than that, users should be following an agile lifecycle of constant upgrade.
-* The Couchbase cluster monitor is proprietary and requires access to the repository to build it into the container. The container can be built without it by removing it and there is a https://github.com/couchbaselabs/observability/tree/main/tools/build-oss-container.sh[helper script^] that does this.
+* CMOS is built for Couchbase Server 7.0 and above.
+  Lower versions are supported but some features might be missing.
+  We intend to improve support for lower versions in future releases, however Couchbase Server 7.0 and above will remain the primary target of development.
+* Limited cross-version compatibility by supporting migrating from previous version to latest version.
+  Best efforts will be made but the intention is this iterates often and no backwards compatibility is provided.
+  We will show how to migrate from X-1 to X but no more than that, users should be following an agile lifecycle of constant upgrade.
+* The Couchbase cluster monitor is proprietary and requires access to the repository to build it into the container.
```

Comment:
```suggestion
* Part of CMOS is the closed-source Couchbase Cluster Monitor, which is included from a private repository at build time.
```

@ingenthr @anilkumar29 Wording change per our chat earlier - how does this sound?

Reply:
CMOS is open source; however, the `Couchbase Cluster Monitor`  is a closed source (private repo) component. If you are building from the source, this component will require access to the repository to make it into the container.

Reply:
```suggestion
* CMOS is open source; however, the Couchbase Cluster Monitor is a closed source (private repository) component.
  If you are building from source, this component will require access to the repository to be included in the container.
```